src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/model/in/CaseUpdate.java

```diff
@@ -5,14 +5,29 @@
 
 public class CaseUpdate {
```

Comment:
I know it's not a new class, so this is just a suggestion: how about renaming it to `CaseUpdateRequest`?

Reply:
ok